scopecheck/test_scopecheck.go

```diff
@@ -0,0 +1,19 @@
+package scopecheck
```

Comment:
test cases are awesome, let's name it `scopecheck_test.go` then it only gets compiled when running `go test ./...`